src/recordlinker/routes/patient_router.py

```diff
@@ -65,6 +66,73 @@ def update_person(
         patient_reference_id=patient.reference_id, person_reference_id=person.reference_id
     )
 
+
+@router.post(
+    "/",
+    summary="Create a patient record and link to an existing person",
+    status_code=fastapi.status.HTTP_201_CREATED,
+)
+def create_patient(
+    payload: typing.Annotated[schemas.PatientCreatePayload, fastapi.Body],
+    session: orm.Session = fastapi.Depends(get_session),
+) -> schemas.PatientRef:
+    """
+    Create a new patient record in the MPI
+    """
+    person = service.get_person_by_reference_id(session, payload.person_reference_id)
+
+    if person is None:
+        raise fastapi.HTTPException(status_code=fastapi.status.HTTP_422_UNPROCESSABLE_ENTITY)
+
+    patient = service.insert_patient(
+        session,
+        payload.record,
+        person=person,
+        external_patient_id=payload.record.external_id,
+        commit=False,
+    )
+    return schemas.PatientRef(
+        patient_reference_id=patient.reference_id, external_patient_id=patient.external_patient_id
+    )
+
+
+@router.patch(
+    "/{patient_reference_id}",
+    summary="Update a patient record",
+    status_code=fastapi.status.HTTP_200_OK,
+)
+def update_patient(
+    patient_reference_id: uuid.UUID,
+    payload: typing.Annotated[schemas.PatientUpdatePayload, fastapi.Body],
+    session: orm.Session = fastapi.Depends(get_session),
+) -> schemas.PatientRef:
+    """
+    Update an existing patient record in the MPI
```

Comment:
For my own clarity, this operation can update the record (the PIIRecord) and/or the `person_reference_id` the existing patient is linked to, right?

Reply:
Correct, this is a `PATCH` operation, therefore every data element is optional.

Reply:
Right. I think the docstring could clarify it with something like "Update an existing patient record in the MPI by updating the contents of the PII Record and/or by updating `person_reference_id` the patient record is linked to." ...or something like that

Reply:
I was thinking the [differences between PUT and PATCH](https://www.geeksforgeeks.org/difference-between-put-and-patch-request/#difference-between-put-and-patch-request) imply that it's an and/or operation, but it doesn't hurt to be explicit.  If you think that's lost and/or confusing, we can certainly expand on the docstring.